instance_skel_types.d.ts

```diff
@@ -176,7 +176,11 @@ export interface CompanionFeedbackBase<TRes> {
 	label: string
 	description: string
 	options: SomeCompanionInputField[]
-	callback?: (feedback: CompanionFeedbackEvent, bank: CompanionBankPNG, info: CompanionFeedbackEventInfo) => TRes
+	callback?: (
```

Comment:
I was referring to this.  Seems like a duplicate of that other one I referenced

Reply:
oh right.
it kinda is a duplicate, but there are 2 ways to define the handler for a feedback. The type could be shared a bit more, but that makes it harder to read so I dont think its beneficial with how little they should be changing